### PR TITLE
Translation Project Wiki

### PR DESCRIPTION
**Pull request by imajit**
Part of #9686 
Added wiki about translation function and importing and exporting translations

### CODE REVIEW

**Review by noi5e — APPROVED** [review]
Thanks for this really helpful documentation!

File: doc/Translation System.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+## Translation life-cycle
+So now that you know all the aspects in this project, let's see how the entire project works. 
+### Step 1: Adding a string
+Add new strings as a key value pair in the `en.yml` file.
+### Step 2: Importing string to Transifex project 
+Transifex automatically fetches new strings from `en.yml` file. Managers and admins can also run manual sync to add strings to Project.
+### Step 3: Translating string
+Translation team members can add translations on the Transifex site. You can find a detailed wiki on how to do so [here](https://publiclab.org/wiki/translation?_=1629559376#Activities+for+people+who+want+to+translate+this+website)
+### Step 4: Exporting translated strings
+Once the string is reviewed,it can be added back to the Github repo. Transifex bot automatically raises a PR once language resource is 100% reviewed. Managers and admins can also trigger a manual sync.
```
Comment (jywarren) [inline doc/Translation System.md:48]: Can we add links and potentially screenshots of where to trigger a manual sync, as well as a link to an example bot PR?

File: doc/Translation System.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+### Exporting translations 
+ Transifex bot automatically raises a PR when a language resource is 100% reviewed. Managers and admins can also trigger a manual sync by `settings->integrations->Send to Github` option, we can specify a threshold percentage and any language files having reviewed percentage above the threshold will be added and a PR will be raised by the Transifex bot. Alternatively, we can also directly download a yaml file for the translation resource from the transifex site. More about it [here](https://publiclab.org/notes/ajitmujumdar25999/07-18-2021/importing-new-translations-from-transifex-project)
+
+## Translation life-cycle
+So now that you know all the aspects in this project, let's see how the entire project works. 
+### Step 1: Adding a string
+Add new strings as a key value pair in the `en.yml` file.
+### Step 2: Importing string to Transifex project 
+Transifex automatically fetches new strings from `en.yml` file. Managers and admins can also run manual sync to add strings to Project.
```
Comment (jywarren) [inline doc/Translation System.md:44]: Is this what I just did at https://www.transifex.com/publiclab/publiclaborg/settings/integrations/ ? Let's link there and also link to https://github.com/publiclab/plots2/pull/10079 as an example!

Comment (imajit) [reply doc/Translation System.md:44]: yes , Sure i'll do that

File: doc/Translation System.md (hunk 1 above)
Comment (imajit) [reply doc/Translation System.md:48]: okay I'll add some screenshots

**Comment by imajit** [comment]
okay I guess i messed the rebase :sweat:

**Comment by jywarren** [comment]
Oops, can we just grab the clean copy of the .md file in `raw` view here and open a new PR? Or did you already? Ah, i see in https://github.com/publiclab/plots2/pull/10114 -- great!

**Pull request closed without merging**